pkg/distro/fedora/package_sets.go

```diff
@@ -182,13 +182,14 @@ func iotCommitPackageSet(t *imageType) rpmmd.PackageSet {
 				"podman-plugins", // deprecated in podman 5
 			},
 		})
-	} else {
-		ps = ps.Append(rpmmd.PackageSet{
-			Include: []string{
-				"bootupd", // added in F40+
-			},
-		})
 	}
+	if common.VersionLessThan(t.arch.distro.osVersion, "41") {
```

Comment:
I think you'd mean `if common.VersionGreaterThanOrEqual(t.arch.distro.osVersion, "41")` here.